pytest_django/plugin.py

```diff
@@ -514,6 +514,20 @@ def _template_string_if_invalid_marker(request):
             else:
                 dj_settings.TEMPLATE_STRING_IF_INVALID.fail = False
 
+
+@pytest.fixture(autouse=True, scope='function')
+def django_clear_site_cache():
```

Comment:
We should prefix this fixture with a `_` to make it clear that it is an internal fixture that is not meant to be requested directly by tests.